pages/index.tsx

```diff
@@ -1,10 +1,53 @@
 import React from "react";
+import Head from "next/head";
+import { PromiseType } from "utility-types";
 import "../styles/style.styl";
 import "../styles/icons.less";
-import Head from "next/head";
 import InviteForm from "../components/InviteForm";
 import Members from "../components/Members";
 import Feed from "../components/Feed";
+import { getChannels } from "../data/channels";
+import emoji from "emoji-dictionary";
+import ReactMarkdown from "react-markdown";
+
+const slackMarkdownRenderer = (text: { value: string }) => {
+  const emojiText = text.value.replace(/:\w+:/gi, (name) =>
+    emoji.getUnicode(name)
```

Comment:
This would fail with custom emoji, wouldn't it? Maybe
```suggestion
    emoji.getUnicode(name) || `:${name}:`
```

or just an empty fallback?

Reply:
Good catch, it would indeed. I'm gonna move all of this emoji stuff to the API as we can have the custom emoji URLs available there